### ISSUE TITLE
Wrong Readme picture

### ISSUE THREAD

**Issue by zaggash** [linuxserver/docker-gsm-csgo#1]
The picture in the readme is the one for TS3.
Here we are talking about CSGO if I'm not wrong :)

**Comment by lonix** [comment]
You are right, but this was a pure copy of the ts3 container, i have not changed much it was about getting csgo working, but not done much :)

**Comment by zaggash** [comment]
Yeah I get it.
I just raise this in order to make it clean :)
